docs/architecture/containerized-lifecycle/toc.yml

```diff
@@ -2,18 +2,16 @@
   href: index.md
   items:
   - name: Introduction to Containers and Docker
-    href: introduction-to-containers-and-docker.md
+    href: ../microservices/container-docker-introduction/index.md?toc=/dotnet/architecture/containerized-lifecycle/toc.json&bc=/dotnet/breadcrumb/toc.json
     items:
     - name: What is Docker?
-      href: what-is-docker.md
+      href: ../microservices/container-docker-introduction/docker-defined.md?toc=/dotnet/architecture/containerized-lifecycle/toc.json&bc=/dotnet/breadcrumb/toc.json
     - name: Docker terminology
-      href: docker-terminology.md
+      href: ../microservices/container-docker-introduction/docker-terminology.md?toc=/dotnet/architecture/containerized-lifecycle/toc.json&bc=/dotnet/breadcrumb/toc.json
     - name: Docker containers, images, and registries
-      href: docker-containers-images-and-registries.md
-    - name: Road to modern applications based on containers
-      href: road-to-modern-applications-based-on-containers.md
```

Comment:
Did you mean to remove this entry completely?

Reply:
Yes, the content was rather small and it didn't seem to add value. It had two links to both e-books that contain the shared content - and brief description of each.